README.md

```diff
@@ -50,49 +50,41 @@ Note that depending on the plugin you add, there might be required permissions a
 repositories {
   mavenCentral()
   maven {
-    url 'https://api.mapbox.com/downloads/v2/releases/maven'
-    authentication {
-      basic(BasicAuthentication)
-    }
-    credentials {
-      username "mapbox"
-      password = "SDK_REGISTRY_TOKEN"
-    }
+    url 'https://api.maplibre.com/downloads/v2/releases/maven'
   }
 }
 
 dependencies {
-  implementation 'com.mapbox.mapboxsdk:{PLUGIN_NAME}-v{MAJOR_MAPS_SDK_VERSION_NUMBER}:PLUGIN_VERSION_NUMBER'
+  implementation 'org.maplibre.android:{PLUGIN_NAME}-v{MAJOR_MAPS_SDK_VERSION_NUMBER}:PLUGIN_VERSION_NUMBER'
```

Comment:
Maybe it would be good to add a concrete example how to add the annotation plugin?

Like so: `implementation 'org.maplibre.gl:android-plugin-annotation-v9:1.0.0'`